pipelines/docker-build-multi-platform-oci-ta/README.md

```diff
@@ -135,6 +135,7 @@ This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/reposito
 ### prefetch-dependencies-oci-ta:0.1 task parameters
 |name|description|default value|already set by|
 |---|---|---|---|
+|ACTIVATION_KEY| Name of secret which contains subscription activation key| activation-key| |
```

Comment:
I think this option is wort mentioning and better describing its usefulness in the commit msg.

Reply:
done